charts/prometheus-snmp-exporter/README.md

```diff
@@ -91,3 +91,19 @@ scrape_configs:
       - target_label: __address__
         replacement: my-service-name:9116  # The SNMP exporter's Service name and port.
 ```
+
+Eaxample configuration via a ServiceMonitor
```

Comment:
imho the servicemonitor could be added as manifest and being enbaled / disabled by config value.
for example like here:
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-rabbitmq-exporter/templates/servicemonitor.yaml

Reply:
Thank you for your commend @monotek. It is actually present here (I also made some fixes in that template down below), and can be enabled/disabled, even my example has the flag in it on line 98